pkg/webhook/v1alpha2/webhook.go

```diff
@@ -84,5 +86,16 @@ func register(manager manager.Manager, server *webhook.Server) error {
 	if err != nil {
 		return err
 	}
-	return server.Register(mutatingWebhook, validatingWebhook)
+	injectWebhook, err := builder.NewWebhookBuilder().
+		Name("mutating.pod.kubeflow.org").
```

Comment:
I think maybe we can rename it to one which is katib related

Reply:
Yes, do you have any suggestion about the name?

Reply:
I think we should follow xx.$group.$domain style (group should be `katib` and domain is `kubeflow.org` here), we'd better make all others consistent with this style.

Reply:
SGTM, it is a historical problem. Can we do the change in another PR? I think we could keep the name here and update it after the PR is merged, with other changes.

Reply:
sure